### ISSUE TITLE
Keyboard Numlock remains dark on bootup.

### ISSUE THREAD

**Issue by ashvinlobo** [BuddiesOfBudgie/budgie-desktop#360]
### Description

Normally when we land on the login screen the numlock lights up. But after i install Fedora 38 Budgie. it's still dark on the login screeen.

### Budgie version

10.7.1

### Operating System

Fedora 38 Budgie Spin

### Steps to reproduce the issue

Tried installing Numlockx and rebooting but it remains off. BIOS settings show numlock on.

### Actual result

Numlock does not light up on the login screen.

### Expected result

Numlock should have lit up on the login screen.

**Comment by serebit** [comment]
By "login screen", you mean LightDM slick-greeter, right? If that's the case, that's either a bug you should file on the [Fedora tracker](https://docs.fedoraproject.org/en-US/quick-docs/howto-file-a-bug/), or the [slick-greeter](https://github.com/linuxmint/slick-greeter) tracker.

**Comment by ashvinlobo** [comment]
Yes exactly what i meant and i sorted this out by adding this line " activate-numlock=true " to the configuration file at
/etc/lightdm/slick-greeter.conf and restarted the desktop. Now all is well and numlock is on on bootup. Prior to this i had installed numlockx which is very important. So closing.
